ckanext/push_errors/tests/test_utils.py

```diff
@@ -0,0 +1,91 @@
+import logging
+from ckanext.push_errors.utils import get_error_trace_id, get_error_log_id
+
+
+class CustomError(Exception):
+    pass
+
+
+def raise_custom_error():
+    """Raises a CustomError to simulate an error scenario for testing purposes."""
+    raise CustomError("Simulated error")
+
+
+def test_get_error_trace_id_returns_path_and_line():
+    """
+    Test that get_error_trace_id returns a string containing the file path and line number
+    where the CustomError was raised, formatted as 'path/to/file.py:<line>'.
+    Ensures the returned trace_id contains a colon and does not end with '.py'.
+    """
+    try:
+        raise_custom_error()
+    except CustomError as e:
+        trace_id = get_error_trace_id(e)
+        assert ":" in trace_id
+        assert not trace_id.endswith(".py")  # it only ends with ":<line>"
+
+
+def test_get_error_trace_id_unknown_when_no_tb():
+    """
+    Test that get_error_trace_id returns 'unknown' when the exception has no traceback.
+
+    This test creates a CustomError instance without a traceback and verifies that
+    the get_error_trace_id function returns the string 'unknown' as expected.
+    """
+    e = CustomError("No traceback")
+    e.__traceback__ = None
```

Comment:
Here we are probably doing that do not exist in real life.
If we have a case in which this is none, test it.
If not, leave it.
```suggestion
```